README.md

```diff
@@ -129,6 +129,8 @@ The scratch storage is probably the more interesting of all the storage options.
 
 PACE recommends that you use Globus to transfer files to and from PACE. I have tried this before, but it is very buggy. I personally recommend using `scp` or `rsync` as ways to transfer files to and from PACE.
 
+Before you use the command to transfer files, make a new instance of the terminal while your main terminal is logged into pace, and then run the rysnc/scp commands on the new terminal. 
```

Comment:
Can you explain why this is needed? Why do you need another terminal on PACE open while running the rsync/scp commands?

Reply:
You don't need to have another terminal. I've done rsync before without having a terminal connected to PACE.